Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -2660,6 +2680,10 @@ extension BrowserViewController: TabDelegate {
         
         tab.addContentScript(WindowRenderHelperScript(tab: tab), name: WindowRenderHelperScript.name(), sandboxed: false)
         
+        let playlistHelper = PlaylistHelper(tab: tab)
+        playlistHelper.delegate = self
+        tab.addContentScript(playlistHelper, name: PlaylistHelper.name(), sandboxed: false)
```

Comment:
Access to `DOM Content` should be available with `sandboxed: true`. I think the scripts can be `sandboxed`, let me know if I'm missing something.

Reply:
Fixed.